packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1582,7 +1582,7 @@ export const components: ThemeOptions["components"] = {
           backgroundColor: theme.palette.grey[50],
           borderColor: theme.palette.grey[300],
 
-          "&::before": {
+          "&.Mui-checked::before": {
             backgroundColor: theme.palette.grey[300],
```

Comment:
Don't we have a color that matches this value? If not, no big. Design Tokens 2.0 will fix it.